Python/Product/PythonTools/PythonTools/Navigation/CodeWindowManager.cs

```diff
@@ -143,8 +145,8 @@ class CodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents {
             var wpfTextView = VsEditorAdaptersFactoryService.GetWpfTextView(vsTextView);
             if (wpfTextView != null) {
                 var factory = ComponentModel.GetService<IEditorOperationsFactoryService>();
-                var editFilter = new EditFilter(wpfTextView, factory.GetEditorOperations(wpfTextView), _serviceProvider);
-                editFilter.AttachKeyboardFilter(vsTextView);
+                EditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
+                ReplEditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
```

Comment:
We're still doing this?

---
In reply to: [61307399](https://github.com/Microsoft/PTVS/pull/1223#discussion_r61307399) [](ancestors = 61307399)